code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -26,6 +26,9 @@ GLOBAL_LIST_EMPTY(PDAs)
 	var/obj/item/cartridge/cartridge = null //current cartridge
 	var/mode = 0 //Controls what menu the PDA will display. 0 is hub; the rest are either built in or based on cartridge.
 	var/icon_alert = "pda-r" //Icon to be overlayed for message alerts. Taken from the pda icon file.
+	var/font_index = 0 //The index of the font array that represents the font selected.
+	var/font_mode = "font-family:\"VT323\", monospace;letter-spacing:1px;" //The currently selected font.

```

Comment:
This shouldn't be initialized here imo, call set_font_mode() or something in Initialize()

Reply:
The program needs these variables as soon as it starts generating HTML. It's what tells the code what the font is.